node/src/test/scala/com/wavesplatform/http/AddressRouteSpec.scala

```diff
@@ -88,6 +88,14 @@ class AddressRouteSpec
     }
 
     r1 shouldNot contain allElementsOf r2
+
+    Get(routePath("/seq/1/9000")) ~> route ~> check {
+      responseAs[JsObject] shouldBe Json.obj("error" -> 10, "message" -> "Too big sequences requested")
```

Comment:
1. `sequences` -> `sequence`
2. Let's tell user in the error text what is the limit